content/influxdb/cloud-dedicated/reference/cli/influxctl/database/create.md

```diff
@@ -72,14 +74,15 @@ influxctl database create [flags] <DATABASE_NAME>
 
 ## Flags
 
-| Flag |                         | Description                                                          |
-| :--- | :---------------------- | :------------------------------------------------------------------- |
-|      | `--retention-period`    | Database retention period (default is 0s or infinite)                |
-|      | `--max-tables`          | Maximum tables per database (default is 500, 0 uses default)         |
-|      | `--max-columns`         | Maximum columns per table (default is 250, 0 uses default)           |
-|      | `--template-tag`        | Tag to add to partition template (can include multiple of this flag) |
-|      | `--template-timeformat` | Timestamp format for partition template (default is `%Y-%m-%d`)      |
-| `-h` | `--help`                | Output command help                                                  |
+| Flag |                         | Description                                                                                                                              |
+| :--- | :---------------------- | :--------------------------------------------------------------------------------------------------------------------------------------- |
+|      | `--retention-period`    | Database retention period (default is 0s or infinite)                                                                                    |
+|      | `--max-tables`          | Maximum tables per database (default is 500, 0 uses default)                                                                             |
+|      | `--max-columns`         | Maximum columns per table (default is 250, 0 uses default)                                                                               |
+|      | `--template-tag`        | Tag to add to partition template (can include multiple of this flag)                                                                     |
+|      | `--template-tag-bucket` | Tag and number of buckets to partition tag values into separated by a comma--for example: `tag1,100` (can include multiple of this flag) |
```

Comment:
Are the double hyphen + no spaces intentional?

Reply:
In the markdown parser we use in the docs, a double-dash renders as an en dash.

- `-` - _(dash)_
- `--` – _(en dash)_
- `---` — _(em dash)_